### PR TITLE
Update to UX job posting

### PR DESCRIPTION
**Pull request by dannychapman**
Revised job posting to better state what we're looking for.

### CODE REVIEW

File: pages/join-user-experience-designer.html [hunk 1]
```diff
@@ -13,13 +13,11 @@
...
     <li>Plan and facilitate user research sessions.</li>
-    <li>Create new online user experiences from existing offline and online experiences.</li>
-    <li>Prototype user flows using mockup tools such as moqups or Balsamic.</li>
-    <li>Perform task analysis and develop resulting task grids.</li>
-    <li>Define mental maps that help identify design opportunities and pain points.</li>
-    <li>Identify and create empathy maps from developed personas and design consistent user journeys and experience maps.</li>
-    <li>Define content models for form design.</li>
-    <li>Create process diagrams and basic wireframes.</li>
+    <li>Work closely with UX research team, stakeholders and developers to determine initial user flows</li>
+    <li>Define new online user experiences from existing paper and in person interactions.</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: I think this has overlap with the preceding and following points

File: pages/join-user-experience-designer.html [hunk 2]
```diff
@@ -13,13 +13,11 @@
...
+    <li>Prototype user flows using mockup tools such as Moqups or Balsamic.</li>
+    <li>Use UX tools (such as content models) to help identify and solve difficult problems.</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: Select and use appropriate UX tools to help identify and solve difficult problems.

File: pages/join-user-experience-designer.html (hunk 1 above)
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: Translate existing paper and in person interactions into new online experiences.

File: pages/join-user-experience-designer.html [hunk 3]
```diff
@@ -13,13 +13,11 @@
...
   <ul>
     <li>Plan and facilitate user research sessions.</li>
-    <li>Create new online user experiences from existing offline and online experiences.</li>
-    <li>Prototype user flows using mockup tools such as moqups or Balsamic.</li>
-    <li>Perform task analysis and develop resulting task grids.</li>
-    <li>Define mental maps that help identify design opportunities and pain points.</li>
-    <li>Identify and create empathy maps from developed personas and design consistent user journeys and experience maps.</li>
-    <li>Define content models for form design.</li>
-    <li>Create process diagrams and basic wireframes.</li>
+    <li>Work closely with UX research team, stakeholders and developers to determine initial user flows</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: oxford comma for lyfe

File: pages/join-user-experience-designer.html [hunk 4]
```diff
@@ -13,13 +13,11 @@
...
+    <li>Define new online user experiences from existing paper and in person interactions.</li>
+    <li>Prototype user flows using mockup tools such as Moqups or Balsamic.</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: Prototype user flows for user testing and internal use.

File: pages/join-user-experience-designer.html [hunk 5]
```diff
@@ -13,13 +13,11 @@
...
+    <li>Use UX tools (such as content models) to help identify and solve difficult problems.</li>
+    <li>Create initial, low fidelity wireframes.</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: Is this redundant to prototyping?

File: pages/join-user-experience-designer.html (hunk 1 above)
Comment (benkutil) [inline pages/join-user-experience-designer.html]: I think this change

```diff
+ existing paper and in person interactions.
```
feels awkward. I’d prefer the original. @dannychapman @goldenmeanie

File: pages/join-user-experience-designer.html [hunk 6]
```diff
@@ -13,13 +13,10 @@
...
+    <li>Work closely with UX research team, stakeholders, and developers to determine initial user flows</li>
     <li>Create new online user experiences from existing offline and online experiences.</li>
-    <li>Prototype user flows using mockup tools such as moqups or Balsamic.</li>
-    <li>Perform task analysis and develop resulting task grids.</li>
-    <li>Define mental maps that help identify design opportunities and pain points.</li>
-    <li>Identify and create empathy maps from developed personas and design consistent user journeys and experience maps.</li>
-    <li>Define content models for form design.</li>
-    <li>Create process diagrams and basic wireframes.</li>
+    <li>Select and use appropriate UX tools to help identify and solve difficult problems.</li>
+    <li>Prototype user flow diagrams and low fidelity wirerames for user testing and internal use.</li>
```
Comment (goldenmeanie) [inline pages/join-user-experience-designer.html]: This feels like it has one too many things in it. Or maybe just shouldn't be last?

**Comment by dannychapman** [comment]
Moved to a fresh PR. Closing.

**Pull request closed without merging**